dags/taar_weekly.py

```diff
@@ -30,6 +37,97 @@
     "taar_weekly", default_args=default_args_weekly, schedule_interval="@weekly"
 )
 
+
+def wipe_args():
+    return [
+        "gsutil",
+        "-m",
+        "rm",
+        "gs://taar_profile_dump/taar-profile.*",
```

Comment:
I use a wildcard to just delete the entire bucket - I think this is ok since the airflow job is the only thing using this bucket.

Reply:
entire bucket would be just gs://taar_profile_dump/* no?